Extensions/ParticleSystem/ExtensionSubDeclaration2.cpp

```diff
@@ -16,30 +16,30 @@ This project is released under the MIT License.
  */
 void ExtensionSubDeclaration2(gd::ObjectMetadata& obj) {
   obj.AddAction("ParticleColor1",
-                _("Initial color"),
-                _("Modify initial color of particles."),
-                _("Put initial color of particles of _PARAM0_ to _PARAM1_"),
+                _("Start color"),
+                _("Modify start color of particles."),
+                _("Change particles start color of _PARAM0_ to _PARAM1_"),
                 _("Common"),
                 "CppPlatform/Extensions/particleSystemicon24.png",
                 "CppPlatform/Extensions/particleSystemicon16.png")
       .AddParameter("object", _("Object"), "ParticleEmitter")
-      .AddParameter("color", _("Initial color"));
+      .AddParameter("color", _("Start color"));
 
   obj.AddAction("ParticleColor2",
-                _("Final color"),
-                _("Modify final color of particles."),
-                _("Put final color of particles of _PARAM0_ to _PARAM1_"),
+                _("End color"),
+                _("Modify end color of particles."),
+                _("Change particles end color of _PARAM0_ to _PARAM1_"),
                 _("Common"),
                 "CppPlatform/Extensions/particleSystemicon24.png",
                 "CppPlatform/Extensions/particleSystemicon16.png")
       .AddParameter("object", _("Object"), "ParticleEmitter")
-      .AddParameter("color", _("Final color"));
+      .AddParameter("color", _("End color"));
 
   obj.AddAction(
          "ParticleRed1",
-         _("Red color, parameter 1"),
-         _("Modify parameter 1 of the red color."),
-         _("the parameter 1 of red color"),
+         _("Red start color"),
```

Comment:
Any chance you could change all references to colors to be "Start color red component"?

Because red/green/blue is a "component" of the overall color :)